project-koku/koku
# Adding year month partitions to Trino temp table SQL

## Jira Ticket

[COST-3666](https://issues.redhat.com/browse/COST-3666)

## Description

This change will add year and month partitions to the Trino/Hive temp tables to prevent multi worker initial ingest inserting/deleting data from different months.

## Testing

1. Checkout Branch
2. Restart Koku
3. Run make customer data
4. See the trino tables have partitions for year and month


## Notes

...
